mne/time_frequency/psd.py

```diff
@@ -112,6 +114,198 @@ def _check_nfft(n, n_fft, n_overlap):
+    --------
+    Raw.plot_psd, Epochs.plot_psd, psd_multitaper
+    """
+    # Prep data
+    data, sfreq = _check_psd_data(inst, tmin, tmax, picks, proj)
+    psds, freqs = _psd_welch(data, sfreq, fmin=fmin, fmax=fmax, n_fft=n_fft,
+                             n_overlap=n_overlap, n_jobs=n_jobs,
+                             verbose=verbose)
+    return psds, freqs
+
+
+@verbose
+def psd_multitaper(inst, fmin=0, fmax=np.inf, tmin=None, tmax=None,
+                   bandwidth=None, adaptive=False, low_bias=True,
+                   normalization='length', picks=None, proj=False,
+                   n_jobs=1, verbose=None):
+    """Compute the power spectral density (PSD) using multitapers.
+
+    Calculates spectral density for orthogonal tapers, then averages them
+    together for each channel/epoch.
+
+    Parameters
+    ----------
+    inst : instance of Epochs or Raw |
+           array of shape ([n_epochs], n_channels, n_times)
+        The data. If not an instance of Epochs or Raw,
+        sfreq must also be supplied.
+    fmin : float
+        Min frequency of interest
+    fmax : float
+        Max frequency of interest
+    tmin : float | None
+        Min time of interest
+    tmax : float | None
+        Max time of interest
+    bandwidth : float
+        The bandwidth of the multi taper windowing function in Hz.
+    adaptive : bool
+        Use adaptive weights to combine the tapered spectra into PSD
+        (slow, use n_jobs >> 1 to speed up computation).
+    low_bias : bool
+        Only use tapers with more than 90% spectral concentration within
+        bandwidth.
+    normalization : str
+        Either "full" or "length" (default). If "full", the PSD will
+        be normalized by the sampling rate as well as the length of
+        the signal (as in nitime).
+    picks : array-like of int | None
+        The selection of channels to include in the computation.
+        If None, take all channels.
+    proj : bool
+        Apply SSP projection vectors. If inst is ndarray this is not used.
+    n_jobs : int
+        Number of CPUs to use in the computation.
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see mne.verbose).
+
+    Returns
+    -------
+    psds : ndarray, shape ([n_epochs], n_channels, n_freqs)
+        The power spectral densities. If Raw is provided,
+        then psds will be 2-D.
+    freqs : ndarray, shape (n_freqs)
+        The frequencies.
+
+    See Also
+    --------
+    Raw.plot_psd, Epochs.plot_psd, psd_welch
+    """
+    # Prep data
+    data, sfreq = _check_psd_data(inst, tmin, tmax, picks, proj)
+    psds, freqs = _psd_multitaper(data, sfreq, fmin=fmin, fmax=fmax,
+                                  bandwidth=bandwidth, adaptive=adaptive,
+                                  low_bias=low_bias,
+                                  normalization=normalization,  n_jobs=n_jobs,
+                                  verbose=verbose)
+    return psds, freqs
```

Comment:
Sorry, I keep noticing small things. Can you simply do 

```py
return _psd_multitaper(data, ...)
```
saves one intermediate step ...